From 4fa8a148132267591bc011a4dcda1666109e1d9d Mon Sep 17 00:00:00 2001
From: Shilin Victor <chrono.monochrome@gmail.com>
Date: Sun, 14 Feb 2016 21:25:34 +0700
Subject: [PATCH 35/95] Revert "ToneGenerator.h: Add C++11 workaround for
 static const float member initialization"

This reverts commit 42f7c42dfe471264e694031d2ac68c4c27152606.

Change-Id: I0833a07fb4563adbb294808c5b519f64244012af
---
 include/media/ToneGenerator.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/media/ToneGenerator.h b/include/media/ToneGenerator.h
index 5ef6201..898b743 100644
--- a/include/media/ToneGenerator.h
+++ b/include/media/ToneGenerator.h
@@ -208,7 +208,7 @@ private:
     static const unsigned int TONEGEN_MAX_WAVES = 3;     // Maximun number of sine waves in a tone segment
     static const unsigned int TONEGEN_MAX_SEGMENTS = 12;  // Maximun number of segments in a tone descriptor
     static const unsigned int TONEGEN_INF = 0xFFFFFFFF;  // Represents infinite time duration
-    #define TONEGEN_GAIN 0.9  // Default gain passed to  WaveGenerator().
+    static const float TONEGEN_GAIN = 0.9;  // Default gain passed to  WaveGenerator().
 
     // ToneDescriptor class contains all parameters needed to generate a tone:
     //    - The array waveFreq[]:
-- 
2.5.0

